app/plugins/ee-rate-limiting.md

```diff
@@ -0,0 +1,34 @@
+---
+id: page-plugin
+title: Plugins - EE Rate Limiting
+header_title: EE Rate Limiting
+header_icon: /assets/images/icons/plugins/rate-limiting.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```

Comment:
The Rate Limiting plugin for Kong Enterprise Edition is a re-engineered version of the incredibly popular Community Edition Rate Limiting plugin, with greatly enhanced configuration options and performance.